### PR TITLE
Remove end of tutorial questions section, give more guidance on exercises

### PR DESCRIPTION
**Pull request by ebatty**

### CODE REVIEW

File: organization/Neuromatch_Tutorial_Format.ipynb [hunk 1]
```diff
@@ -69,19 +69,21 @@
...
+        "Use a line (---) separator from title block to objectives. You should briefly introduce your content here in a few sentences. In this tutorial, you will learn what an NMA tutorial notebook should look like. \n",
         "\n",
-        "Then have a few specific objectives for this tutorial:\n",
+        "Then have a few specific objectives for this tutorial, ideally [SMART goals](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944406/#:~:text=1%20The%20mnemonic%20SMART%E2%80%94Specific,a%20well%2Dwritten%20learning%20objective.&text=Verbs%20such%20as%20understand%2C%20know,measurable%20and%20should%20be%20avoided.). By the end of this tutorial you will be able to:\n",
         "*   Understand the format of an NMA tutorial\n",
-        "*   Learn about code exercises, interactive demos, and group discussions\n",
+        "*   Make code exercises, interactive demos, and group discussions the correct structure\n",
+        "\n",
+        "**For production editors: make sure to clear outputs before pushing to git**\n",
```
Comment (mwaskom) [inline organization/Neuromatch_Tutorial_Format.ipynb:78]: CI handles this?

Comment (ebatty) [reply organization/Neuromatch_Tutorial_Format.ipynb:78]: the clearing outputs part? ah I didn't realize that

Comment (mwaskom) [reply organization/Neuromatch_Tutorial_Format.ipynb]: https://github.com/NeuromatchAcademy/course-content/blob/master/ci/process_notebooks.py#L254-L299

Comment (mwaskom) [reply organization/Neuromatch_Tutorial_Format.ipynb]: (Though if DL is not going to use the CI, they'll need to remember to do this)

**Pull request merged**